src/redis-cli.c

```diff
@@ -2735,12 +2735,15 @@ static int isSensitiveCommand(int argc, char **argv) {
         return 1;
     } else if (argc > 2 &&
         !strcasecmp(argv[0],"config") &&
-        !strcasecmp(argv[1],"set") && (
-            !strcasecmp(argv[2],"masterauth") ||
-            !strcasecmp(argv[2],"masteruser") ||
-            !strcasecmp(argv[2],"requirepass")))
-    {
-        return 1;
+        !strcasecmp(argv[1],"set")) {
```

Comment:
Should we add a tcl test for this?

Reply:
Yeah, I don't think we have any, might be a good time to add to avoid regressions in the future.

Reply:
Let me try it

Reply:
i'm ok to leave this untested. i don't think there's a good ROI for such test.